src/components/Header/Header.tsx

```diff
@@ -0,0 +1,25 @@
+import { Flex } from "@adobe/react-spectrum";
+import { ToggleButton } from "@adobe/react-spectrum";
+import ClassicGridView from "@spectrum-icons/workflow/ClassicGridView";
+import React, { FC } from "react";
```

Comment:
Aquí agreguemos el `useState` method a la lista de imports para poder usarlo directamente sin tener que type `React.useState`

```suggestion
import React, { FC, useState } from "react";
```